dwhdelft.nl/components/pages/resources/PurpleFriday.vue

```diff
@@ -0,0 +1,62 @@
+<i18n lang="yaml">
+en:
+  title: 'Purple Friday 2024'
+  subtitle: 'What is happening on Purple Friday this year? Find an overview here:'
+nl:
+  title: 'Paarse Vrijdag 2024'
+  subtitle: 'Wat is er dit jaar op Paarse Vrijdag? Zie hier een overzicht (alleen in het Engels):'
+</i18n>
+
+<script setup>
+import { ref, watchEffect } from 'vue'
+const { t } = useT()
+
+const faculties = (await useAsyncData(() => queryContent('resources').find())).data
+const active = ref(0)
+
+function setActive(index) {
+  active.value = index
+}
+
+watchEffect(() => {
+  faculties.value.sort((a, b) => {
+    return (a.order || 0) - (b.order || 0)
+  })
+})
+</script>
+
+<template>
+  <div class="w-full py-6">
+    <section class="container mx-auto space-y-6">
+      <div class="space-y-6">
+        <div class="text-5xl">
+          {{ t('title') }}
+        </div>
+        <div>
+          {{ t('subtitle') }}
+        </div>
+        <div class="text-2xl">
+          <div class="lg:w-[36rem] xl:w-[44rem] pt-12">
+            <div class="flex space-x-2 overflow-auto whitespace-nowrap">
+              <div
+                v-for="(faculty, index) in faculties"
+                :key="faculty.name"
+                :class="index === active ? 'bg-brand-800' : 'bg-brand-500 cursor-pointer hover:bg-brand-400'"
+                class="text-white rounded-t-lg p-4 font-semibold tracking-wide z-20 relative"
+                @click="setActive(index)"
```

Comment:
```suggestion
                @click="active = index"
```
kan ook, dan heb je die functie niet nodig (maar dit is ook fine)